source/content/supported-drupal.md

```diff
@@ -40,16 +40,84 @@ terminus site:create <site> <label> drupal8
+  * Or a Custom Upstream based on the Pantheon Drupal 7 Upstream with the [latest updates pulled in](/guides/custom-upstream/create-custom-upstream#pull-in-core-from-pantheons-upstream).
+* Sites must be updated to the latest release of Drupal core (currently 7.101).
+<br/>
+<br/>
+
+To verify your current Drupal core version, you can check the [Status report tab](/drupal-launch-check#status-report) in the Pantheon site dashboard or you can use Terminus: 
+
+```bash{promptUser: user}
+terminus drush <site>.<env> status
+```
+
+To verify your current upstream, you can go to the Pantheon site dashboard and click **Settings**, then **About Site** or you can use Terminus: 
+
+```bash{promptUser: user}
+terminus site:info <site>
+```
+
+If you are not already using the Pantheon Drupal 7 upstream, you can [switch the site upstream via Terminus](/terminus/examples#switch-upstreams).
+
+#### D7 Contrib Modules LTS
+This includes security patches and compatibility updates to community-contributed modules that power their site via Tag1’s [D7 Extended Support (D7ES) module](https://www.drupal.org/project/tag1_d7es), which will be distributed and maintained as part of the Pantheon Drupal 7 Upstream.
+
+##### Requirements
+* Available to all site plans for sites associated with a Gold, Diamond, or Platinum [workspace account tier](/guides/account-mgmt/plans/workspace-plans).
+* Sites must use the [Pantheon Drupal 7 Upstream](https://github.com/pantheon-systems/drops-7).
+  * Or a Custom Upstream based on the Pantheon Drupal 7 Upstream with the [latest updates pulled in](/guides/custom-upstream/create-custom-upstream#pull-in-core-from-pantheons-upstream).
+* Sites must be updated to the latest release of Drupal core (currently 7.101).
+* (Recommended) Not a requirement, but we strongly recommend updating all contrib modules to their latest versions as well. 
+
+##### D7ES Usage
+By enabling this module, your site’s list of contrib modules will be automatically submitted for D7ES security coverage. You will receive email notifications of D7 core & installed contrib module security and compatibility updates. 
+
+Enable the module, then go to `/admin/config/system/tag1-d7es` and configure the list of email addresses that should be notified when applicable updates are released. On this configuration page you can also confirm your sites connection with Tag1’s update servers.
+
+Apply contrib updates from the Tag1 D7ES service via Drush, Autopilot, or manually via SFTP. For details, see the following tabs.
+
+<TabList>
+
+<Tab title="Drush (Recommended)" id="drush" active={true}>
+
+1. [Set the connection mode to SFTP](/guides/sftp) for the Dev or Multidev environment via the Pantheon Dashboard or with [Terminus](/terminus):
+
+ ```bash{promptUser: user}
+ terminus connection:set <site>.<env> sftp
+ ```
+
+1. Use Drush to apply contrib updates from the Tag1 D7ES service: 
+ ```bash{promptUser: user}
+ terminus connection:set <site>.<env> pm:update --no-core
```

Comment:
@scottbuscemi can you confirm whether `--no-core` is needed here? The gdoc did not have this flag but I think it's needed so that our upstream core files aren't overwritten

Reply:
Yes. This file should prevent core from being overwritten:

https://github.com/pantheon-systems/drops-7/blob/default/.drush-lock-update

I don't recall whether you get an error or warning if `--no-core` is omitted, but it's best to include it for clarity.